hw8Task1.js

```diff
@@ -0,0 +1,46 @@
+const numberArr = [7, 8, 2, 30, 85, 95, 77, 94, 37, 31]
+// TASK 1
+numberArr.forEach((value) => {
+    if(value % 3 == 0){
+        console.log(value)
+    }
+})
+// TASK 2
+const mapArray = numberArr.map((value, index, numberArr) => {
```

Comment:
```suggestion
const mapArray = numberArr.map((value, index, arr) => {
```
Можно называть иначе, чтобы самому не путаться)